build/amis/ansible/roles/common/tasks/redhat.yml

```diff
@@ -15,6 +15,8 @@
 - name: add epel repo
   yum:
     name: "{{ common_redhat_epel_rpm }}"
+    state: present
+    lock_timeout: 60
```

Comment:
lock_timeout is needed for Ansible version 2.8.0+, and only available in version 2.8.0+, which prompted the previous version requirement bump for Ansible in build/amis/README.md